gulpfile.js

```diff
@@ -1,125 +1,82 @@
-});
-
-gulp.task('fonts', gulp.series('fonts:clean', 'fonts:copy'));
-
-gulp.task('fonts:watch', function () {
-	gulp.watch('assets/fonts/**/*', watchOpt, gulp.parallel('fonts'));
-});
-
-gulp.task('build:clean', function () {
-	return del(['dist']);
-});
-
-//gulp.task('assets', gulp.series('svg', gulp.parallel('css', 'js:lint', 'js', 'fonts', 'images')));
-gulp.task('assets', gulp.series(gulp.parallel('css', 'fonts')));
-
-//gulp.task('watch', gulp.parallel('svg:watch', 'css:watch', 'js:watch', 'fonts:watch', 'images:watch'));
-gulp.task('watch', gulp.parallel('css:watch', 'fonts:watch'));
-
-gulp.task('dev', gulp.series('assets', 'fractal:start', 'watch'));
-
-gulp.task('build', gulp.series(function (cb) {
-	isProduction = true;
-	cb();
-}, 'assets', 'build:clean', 'fractal:build'));
 
-gulp.task('default', gulp.series('build'));
+/* 
+We have two goals currently:
+
+1. generate & update (public/stylesheets/site.css) whenever a component scss file is updated 
+    so we can reference it within (components/_preview.hbs) for a real-time preview.
+2. start fractal server and let it do its thing (watch, livereload, build)
+*/
+
+// copy generated.scss with dynamic imports for
+// all component sass files into public/css   so
+// we can use it in /components/_preview.hbs
+function styles() {
+	return src('src/scss/**/*.scss')
+    .pipe(plumber())
+    .pipe(sassGlob())
+    .pipe(sass.sync({ outputStyle: 'expanded', precision: 10 }).on('error', sass.logError))
+    .pipe(postcss([autoprefixer()]))
+	// .pipe(postcss([cssnano({safe: true, autoprefixer: false})]))
+	// .pipe(gulpif(isProd, postcss([cssnano({safe: true, autoprefixer: false})])))
+    .pipe(rename('site.css'))
+    .pipe(dest('public/stylesheets', { sourcemaps: true, }));
+}
+
+function images() {
+	return src('public/images/**/*', { since: lastRun(images) })
+    .pipe(imagemin())
+    .pipe(dest('dist/images'));
+}
+
+function fonts() {
+	return src('public/fonts/**/*.{eot,svg,ttf,woff,woff2}')
+    .pipe(dest('dist/fonts'));
+}
+
+// We really don't need clean() due to ephemeral philosophy of build systems.
+// Build assets are meant to be temporary, so 'overwrite' is default
+// - Fractal deletes dist/ on start()
+// - Gulp overwrites /public/stylesheets/site.css
+// - We can add dist/ etc.. to gitignore if we don't want to commit them
+// - If piping or cache becomes an issue, then clean() becomes useful.
+function clean() {
+	return del(['dist'])
+}
+
+async function startFractal() {
+	// rebuild site.css onSave
+    watch('components/**/*.scss', styles);
+	//watch('public/js/**/*.js', scripts);
+
+	const server = fractal.web.server({ sync: true });
+		  server.on('error', err => logger.error(err.message));
+
+	await server.start();
+	logger.success(`Fractal server is now running at ${server.url}`);
+}
+
+exports.styles = series(styles); // `npm run styles` OR `gulp styles`
+exports.default = series(clean, styles, startFractal); // `npm run start` OR `gulp`
```

Comment:
reading this, it looks like images and fonts is no longer called, will this break the one test image we have in the repo right now (content card example)?

Reply:
Agghh - lol - that was sloppy of me.

The Gulp tasks for images() and font() was for meant for a "watch" that I failed to include in the default task.

Fractal handles all build  and generation of images and fonts, but I'm not sure if optimizations are applied to images and font files.

Now that I think of it, maybe we should keep these tasks in place to apply optimizations before Fractal grabs them for the build.

I'll add them to the default task.